dashboard/src/actions/apps.ts

```diff
@@ -81,25 +81,36 @@ function getAppUpdateInfo(
   appVersion: string,
 ): ThunkAction<Promise<void>, IStoreState, null, AppsAction> {
   return async dispatch => {
-    const chartsInfo = await Chart.listWithFilters(chartName, currentVersion, appVersion);
-    let updateInfo: IChartUpdateInfo = {
-      repository: { name: "", url: "" },
-      latestVersion: "",
-    };
-    chartsInfo.forEach(c => {
-      const chartLatestVersion = c.relationships.latestChartVersion.data.version;
-      if (semver.gt(chartLatestVersion, currentVersion)) {
-        if (updateInfo.latestVersion && semver.gt(updateInfo.latestVersion, chartLatestVersion)) {
-          // The current update is newer than the chart version, do nothing
-        } else {
-          updateInfo = {
-            latestVersion: chartLatestVersion,
-            repository: c.attributes.repo,
-          };
-        }
+    dispatch(requestApps());
+    try {
+      const chartsInfo = await Chart.listWithFilters(chartName, currentVersion, appVersion);
+      let updateInfo: IChartUpdateInfo = {
+        upToDate: true,
```

Comment:
previously, the condition to discern if an app has an update was to check if `latestVersion` wasn't empty. Now there is an explicit parameter for that since `latestVersion` will be populated in most of the cases, even if it's equal to the current version.